static/js/app.js

```diff
@@ -401,6 +401,46 @@ window.onerror = function reportClientException(message, source, line_number, co
   });
 }
 
+function convert_lang_to_ISO(lang){
+switch (lang){
+    case 'nl':
+        return 'nl-NL';
+    case 'hu':
+        return 'hu-HU';
+    case 'sw':
+        return 'sw-KE';
+    case 'fr':
+        return 'fr-FR';
+    case 'pt':
+        return 'pt-BR';
+    case 'de':
+        return 'de-DE';
+    case 'it':
+        return 'it-IT';
+    case 'el':
+        return 'el-GR';
+    case 'zh':
+        return 'zh-CN';
+    case 'es':
+        return 'es-ES';
+    default:
+        return 'en-US';
+
+}
+}
+
+function speak(text) {
+    speak_enabled = $('#speak').is(":checked")
+    console.log('speak?', speak_enabled);
+    if (speak_enabled){
```

Comment:
```suggestion
    if (speak_enabled && window.SpeechSynthesisUtterance) {
```

Otherwise this will crash with a `ReferenceError` if run on a browser that doesn't have the speech API.